Sparkle/SUUpdateAlert.m

```diff
@@ -107,15 +107,10 @@ - (IBAction)remindMeLater:(id)__unused sender
 
 - (void)displayReleaseNotes
 {
-    // Set the default font
     [self.releaseNotesView setPreferencesIdentifier:SUBundleIdentifier];
     WebPreferences *prefs = [self.releaseNotesView preferences];
-    NSString *familyName = [[NSFont systemFontOfSize:8] familyName];
-    if ([familyName hasPrefix:@"."]) { // 10.9 returns ".Lucida Grande UI", which isn't a valid name for the WebView
-        familyName = @"Lucida Grande";
-    }
-    [prefs setStandardFontFamily:familyName];
-    [prefs setDefaultFontSize:(int)[NSFont systemFontSizeForControlSize:NSSmallControlSize]];
+    [prefs setStandardFontFamily:@"-apple-system-font"];
```

Comment:
OK... please add a comment explaining the `-apple-system` and `-apple-system-font` tokens and why we're using it.